tests/database/database-manager.test.ts

```
@@ -0,0 +1,353 @@
+      .mockResolvedValueOnce({ rows: [{ '1': 1 }], columns: ['1'] }); // SELECT 1 (verifyConnection)
+
+    mockCreateClient.mockReturnValue({ execute: mockExecute, close: vi.fn() });
+    mockRunMigrations.mockResolvedValue();
+
+    // Mock console methods
+    vi.spyOn(console, 'error').mockImplementation(() => {});
+    vi.spyOn(console, 'log').mockImplementation(() => {});
+
+    // Reset environment variables
+    process.env = { ...originalEnv };
+    delete process.env.DATABASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)